lib/update/phase_app.go

```diff
@@ -187,34 +186,31 @@ func (p *phaseApp) runHooks(ctx context.Context, hooks ...schema.HookType) error
 		_, err := app.CheckHasAppHook(p.Apps, req)
 		if err != nil {
 			if trace.IsNotFound(err) {
-				log.Debugf("%v does not have %v hook.", p.Package, hook)
+				p.Debugf("%v does not have %v hook.", p.Package, hook)
```

Comment:
This cannot use `WithFields` as `fsm.Logger` cannot fully implement `logrus.FieldLogger` - namely, `WithXXX` methods are supposed to return `*logrus.Entry` instead of expected `logrus.FieldLogger`